pom.xml

```diff
@@ -0,0 +1,134 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <name>OpenRefine - FAIR Metadata extension</name>
+    <description>Extension for managing metadata in FAIR way</description>
+    <url>https://github.com/FAIRDataTeam/OpenRefine-metadata-extension</url>
+
+    <groupId>org.fair.openrefine</groupId>
+    <artifactId>metadata</artifactId>
+    <packaging>jar</packaging>
+    <version>0.1.0-SNAPSHOT</version>
+
+    <repositories>
+        <repository>
+            <id>maven-repository</id>
+            <url>file:///${project.basedir}/project-repository</url>
+        </repository>
+    </repositories>
+
+    <build>
+        <finalName>metadata</finalName>
+        <resources>
+            <resource>
+                <directory>src/main/java</directory>
```

Comment:
This is redundant, since it defaults to `src/main/java`.

Reply:
Fixed in https://github.com/FAIRDataTeam/OpenRefine-metadata-extension/commit/8c5871ba9e3ed2c2e3324327e6da3560dd81c5c3